client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useStocktakeLineEdit.ts

```diff
@@ -35,6 +44,58 @@ export const useStocktakeLineEdit = (
       })
     );
 
+  const save = async () => {
+    let result;
```

Comment:
I like early returns. 
All the same - this syntax we haven't used elsewhere, and we've tried to avoid mutable variables, so I would put the whole `save` function body inside the `try` block and have `const result = await mutateAsync(draftLines);`

Reply:
Changed it here: e58e8dc9cf3f2b8e680a0954fd6e90165ae51729